pkg/config/v1alpha1/metadata_types.go

```diff
@@ -21,6 +21,7 @@ type Metadata struct {
 
 type MetadataSpec struct {
 	PastMirrors []PastMirror `json:"pastMirrors"`
+	Files       []File       `json:"files"`
```

Comment:
What are these files for? Can you add a comment (I was bad about adding comments when I updated these types, excuse my hypocrisy :grin:)

Reply:
Changed to PastFiles to keep the naming consistent and added comments for that and PastMirrors.